str/helper/merge_str_vcf_combiner.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+"""
+This script combines sharded output VCFs from mergeSTR into one VCF, and assumes the genotyper used was ExpansionHunter.
+
+analysis-runner --access-level test --dataset tob-wgs --description \
+    'VCF combiner' --output-dir 'hoptan-str/shard_workflow_test/merge_str_vcf_combiner' \
+    merge_str_vcf_combiner.py \
+    --input-dir=gs://cpg-tob-wgs-test/hoptan-str/shard_workflow_test/merge_str
+"""
+import gzip
+import click
+
+from cpg_utils.config import get_config
+from cpg_utils import to_path
+from cpg_utils.hail_batch import get_batch, output_path
```

Comment:
```suggestion
from cpg_utils.hail_batch import output_path
```

I don' think there's any need for a nested batch?